src/components/Layout/index.tsx

```diff
@@ -70,7 +72,12 @@ const navigation = [
 
 const Layout: React.FC = ({ children }) => (
   <Wrapper>
-    <LegalBanner title="💀 This project is in beta. Use at your own risk." />
+    <LegalBanner>
+      <p>
+        <FontAwesomeIcon icon={faSkull} style={{ marginRight: '0.3rem' }} /> This project is in beta. Use at your own
```

Comment:
Probably could just make this hardcoded inside `LegalBanner` but i like to make things more complicated for myself i guess